### PR TITLE
Update golangci lint config

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

This updates the golangci lint config to stop ignoring all revive issues so we can fix a couple of them. More fix PRs coming soon, trying to keep PRs small. 

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/users/users_embed.go [hunk 1]
```diff
@@ -5,15 +5,19 @@ import (
 )
 
 var (
+	// EmptyExample ...
```
Comment (Widcket) [inline internal/users/users_embed.go:8]: Let's complete these.

Comment (sergiught) [reply internal/users/users_embed.go:8]: I was actually thinking about bringing the internal/users package inside the internal/cli/data pkg and then make these private, as they're pretty self explanatory, wdyt?

Comment (sergiught) [reply internal/users/users_embed.go]: I completed the comments for now tho: [`014023f` (#669)](https://github.com/auth0/auth0-cli/pull/669/commits/014023f9e68a72e4151559f8d9fdc8ea942dd3bc).

**Review by willvedd — APPROVED** [review]

File: internal/users/users_embed.go (hunk 1 above)
Comment (Widcket) [reply internal/users/users_embed.go]: > I was actually thinking about bringing the internal/users package inside the internal/cli/data pkg and then make these private, as they're pretty self explanatory, wdyt?

Yeah, no need to have them be public.

**Review by Widcket — APPROVED** [review]

**Pull request merged**